cilium/cmd/bpf_policy_get.go

```diff
@@ -141,25 +141,40 @@ func formatMap(w io.Writer, statsMap []policymap.PolicyEntryDump) {
 	}
 
 	if printIDs {
-		fmt.Fprintf(w, "%s\t%s\t%s\t%s\t%s\t%s\t%s\t\n",
-			policyTitle, trafficDirectionTitle, labelsIDTitle, portTitle, proxyPortTitle, bytesTitle, packetsTitle)
+		fmt.Fprintf(w, "%s\t%s\t%s\t%s\t%s\t%s\t%s\t%s\t\n",
+			policyTitle, trafficDirectionTitle, labelsIDTitle, portTitle, proxyPortTitle, bytesTitle, packetsTitle, prefixTitle)
 	} else {
-		fmt.Fprintf(w, "%s\t%s\t%s\t%s\t%s\t%s\t%s\t\n",
-			policyTitle, trafficDirectionTitle, labelsDesTitle, portTitle, proxyPortTitle, bytesTitle, packetsTitle)
+		fmt.Fprintf(w, "%s\t%s\t%s\t%s\t%s\t%s\t%s\t%s\t\n",
+			policyTitle, trafficDirectionTitle, labelsDesTitle, portTitle, proxyPortTitle, bytesTitle, packetsTitle, prefixTitle)
 	}
 	for _, stat := range statsMap {
+		prefixLen := stat.Key.Prefixlen - policymap.GetStaticPrefixBits()
 		id := identity.NumericIdentity(stat.Key.Identity)
 		trafficDirection := trafficdirection.TrafficDirection(stat.Key.TrafficDirection)
 		trafficDirectionString := trafficDirection.String()
-		port := models.PortProtocolANY
-		if stat.Key.DestPort != 0 || stat.Key.Nexthdr == uint8(u8proto.ICMP) || stat.Key.Nexthdr == uint8(u8proto.ICMPv6) {
-			dport := byteorder.NetworkToHost16(stat.Key.DestPort)
-			proto := u8proto.U8proto(stat.Key.Nexthdr)
-			port = fmt.Sprintf("%d/%s", dport, proto.String())
+
+		port := models.PortProtocolANY // Wildcard protocol and dport
+		if prefixLen > 0 {
+			if prefixLen < 8 || prefixLen > 24 {
+				port = fmt.Sprintf("<INVALID PORT PREFIX LENGTH: %d>", prefixLen)
+			} else {
+				protoStr := u8proto.U8proto(stat.Key.Nexthdr).String()
+				if prefixLen == 8 {
+					port = protoStr
+				} else {
+					dport := stat.Key.GetDestPort()
+					if prefixLen < 24 {
+						port = fmt.Sprintf("0x%x/%d/%s", dport, prefixLen-8, protoStr)
+					} else {
+						port = fmt.Sprintf("%d/%s", dport, protoStr)
+					}
+				}
+			}
```

Comment:
I believe this is the core change of the PR that lays the foundation for protocol/port wildcarding. Can we have a switch case here to decide which field(s) are wildcarded, and annotate the cases with "protocol wildcarded (prefix len == 8), etc"? 
Also, can we not use hardcoded values here (how about using `sizeof`)?

Reply:
Refactored this to be done once in policymap package, in the form suggested here (switch case with comments, using consts defined with Sizeof instead of hard coding the field sizes.